data/src/test/java/org/apache/iceberg/data/TestGenericReaderDeletes.java

```diff
@@ -18,28 +18,23 @@
  */
 package org.apache.iceberg.data;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 import java.io.File;
 import java.io.IOException;
-import java.nio.file.Files;
-import org.apache.iceberg.ParameterizedTestExtension;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.TestTables;
 import org.apache.iceberg.io.CloseableIterable;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 import org.apache.iceberg.util.StructLikeSet;
-import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.Assert;
 
-@ExtendWith(ParameterizedTestExtension.class)
 public class TestGenericReaderDeletes extends DeleteReadTests {
 
   @Override
   protected Table createTable(String name, Schema schema, PartitionSpec spec) throws IOException {
-    File tableDir = Files.createTempDirectory(temp, "junit").toFile();
```

Comment:
please revert changes to this file as this file has been already converted to JUnit5 + AssertJ